app/templates/weight.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% block content %}
+<form method="POST" action="/weight/">
+    <div class="mb-3 weight target_weight"><br>
+        <label for="target">Target</label><br>
```

Comment:
you can add margin instead of using `<br>`

Reply:
+1